app/views/layouts/application.html.erb

```diff
@@ -25,7 +25,7 @@
   </head>
 
   <body class="h-full">
-    <div id="notification-tray" class="fixed z-50 space-y-1 top-6 right-6"></div>
+    <div id="notification-tray" class="fixed z-50 space-y-1 top-6 right-10"></div>
```

Comment:
Based on Figma, we should really move this further to the left, so it's visually inside the app container.  On my box, this was the highest class available.  Does anyone know why larger numbers are not available to us, or is it just my box?